source/patterns/@aws-solutions-constructs/aws-lambda-elasticsearch-kibana/test/lambda-elasticsearch-kibana.test.ts

```diff
@@ -133,48 +133,34 @@ test('check override cognito domain name with provided cognito domain name', ()
   });
 });
 
-test('Check for 3 subnets for environment specified stack', () => {
-  const stack = new cdk.Stack(undefined, undefined, {
-    env: { account: "123456789012", region: 'us-east-1' },
-  });
+test("Test minimal deployment that deploys a VPC in 2 AZ without vpcProps", () => {
```

Comment:
For environment-agnostic stacks (account not defined), it deploys a VPC in 2 availability zones since there are regions that currently have only 2 availability zones, so that is the safe bet for the CDK.